backend/config.py

```diff
@@ -17,12 +17,14 @@ class DevelopmentConfig(Config):
     TESTING = True
 
     """db config"""
-    SQLALCHEMY_DATABASE_URI = "postgresql://{username}:{password}@{hostname}:{port}/{databasename}".format(
-        username=os.environ.get("DB_USER"),
-        password=os.environ.get("DB_PASSWORD"),
-        hostname=os.environ.get("DB_HOST"),
-        port=5432,
-        databasename=os.environ.get("DB_NAME"),
+    DB_USER = (os.environ.get("DB_USER"),)
+    DB_PASSWORD = (os.environ.get("DB_PASSWORD"),)
+    DB_HOSTNAME = (os.environ.get("DB_HOST"),)
+    DB_PORT = (5432,)
+    DB_NAME = os.environ.get("DB_NAME")
+
+    SQLALCHEMY_DATABASE_URI = (
```

Comment:
Why is it wrapped around with braces `(<value>)` ?
They will be omitted by the interpreter but still seems bit odd. Is it because the line is too long?